lib/pingurl.js

```diff
@@ -1,8 +1,10 @@
 'use strict';
 
 const url = require('url');
-const http = require('http');
-const https = require('https');
+const followRedirects = require('follow-redirects');
+followRedirects.maxRedirects = 10;
```

Comment:
quick google shows that browsers have between 10 (IE) and 20 (chrome), so this is a fine assumption.